saleor/graphql/product/tests/mutations/test_product_update.py

```diff
@@ -1462,7 +1462,7 @@ def test_update_product_with_no_id(
     assert data["errors"] == [
         {
             "field": "attributes",
-            "code": ProductErrorCode.REQUIRED.name,
+            "code": ProductErrorCode.INVALID.name,
```

Comment:
Are we changing an error code? I think it might be a breaking change in some cases.